Form/Type/ButtonIconType.php

```diff
@@ -14,7 +15,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder->add(
             'example',
-            'submit',
+            SubmitType::class,
```

Comment:
This is deprecated and will be removed in 3.0, see https://github.com/symfony/symfony/blob/master/UPGRADE-3.0.md#form